abi/abi.go

```diff
@@ -187,6 +187,16 @@ func Uint64(i uint64) Item {
 	return Item{Type: abit.Uint64, d: b[:]}
 }
 
+func Uint8(i uint8) Item {
+	var b [32]byte
+	bint.Encode(b[:], uint64(i))
```

Comment:
This should be fine since you are converting an 8-bit uint to a 64-bit uint. There is no data loss.